nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/provenance-event-dialog/provenance-event-dialog.component.html

```diff
@@ -17,6 +17,7 @@
 
 <h2 mat-dialog-title>Provenance Event</h2>
 <div class="provenance-event">
+    <error-banner></error-banner>
```

Comment:
We might need to devise some way of giving these banners some context. I found a scenario where I had a banner error while on the lineage screen then opened the View Details option of one of the nodes and now the error is in 2 places at the same time. Dismissing one, dismisses them both.

<img width="1729" alt="Screenshot 2024-02-13 at 1 43 47 PM" src="https://github.com/apache/nifi/assets/713866/edd94d1f-d187-4a7d-900b-b32ee7fac53d">


Maybe we need to specify the source of the banner error in the store (optionally???) when firing them, and then the banner elements can take an input of the context they should be interested in?

```
<error-banner context="provenance-event-diaglog" ></error-banner>
```

Reply:
I meant to delete this `error-banner` because there was no scenario where errors could surface from the dialog directly. If I do, we could punt the contextual banner errors down the road until we hit it again.